frontend/src/concepts/pipelines/content/import/PipelineVersionImportModal.tsx

```diff
@@ -47,6 +58,42 @@ const PipelineVersionImportModal: React.FC<PipelineVersionImportModalProps> = ({
     resetData();
   };
 
+  const onSubmit = () => {
+    setImporting(true);
+    setError(undefined);
+
+    if (uploadOption === PipelineUploadOption.FILE_UPLOAD) {
+      api
+        .uploadPipelineVersion({}, name, description, fileContents, pipelineId)
+        .then((pipelineVersion) => onBeforeClose(pipelineVersion, pipeline))
+        .catch((e) => {
+          setImporting(false);
+          setError(e);
+        });
+    } else {
+      api
+        .createPipelineVersion({}, pipelineId, {
+          // eslint-disable-next-line camelcase
```

Comment:
Same comment as in `PipelineImportModal.tsx`.

Which makes me wonder if we need to make these underscore variable names only trigger warnings instead of errors. Seems like we are constantly having to disable linting when accessing basically anything from an API data source. 

Ideally, we could create a rule that allows usage of non-camelcase variables, but not the ability to define new non-camelcase variables.

Reply:
We can open a tech debt issue on this, but for now i will take your suggestion